el-patch.el

```diff
@@ -579,10 +630,16 @@ patched. NAME, TYPE, and VARIANT are as returned by
 `el-patch-get'."
   (interactive (el-patch--select-patch))
   (if-let ((patch-definition (el-patch-get name type variant)))
-      (eval `(el-patch--stealthy-eval
-              ,(el-patch--resolve-definition
-                patch-definition nil)
-              "This function was patched and then unpatched by `el-patch'."))
+      (if (car variant)
+          ;; an advice, remove it
+          (advice-remove name
+                         (el-patch--advice-name name (cdr variant)))
```

Comment:
Note that an advice takes precedence regardless, even if a patch was applied after it. It must be manually removed or un-patched.